tests/test_scale.py

```diff
@@ -1,40 +1,30 @@
+from visumorph import scale, load_image
 import numpy as np
-import visumorph as vm
 import pytest
-from PIL import Image
 
 
-def test_scale_can_produce_a_larger_image():
-    img = vm.load_image("tests/img/raw/meme.jpg")
-    original_size = np.array(img.get_dimensions()[:2])
-    resized = vm.scale(img, 2.0)
-    new_size = resized.get_dimensions()[:2]
-    assert np.array_equal(new_size, original_size * 2), \
-        "The scaled image does not have the expected dimensions!"
-
-
-def test_scale_can_produce_a_smaller_image():
-    img = vm.load_image("tests/img/raw/meme.jpg")
-    original_size = np.array(img.get_dimensions()[:2])
-    resized = vm.scale(img, 0.01)
-    new_size = resized.get_dimensions()[:2]
-    assert np.array_equal(new_size, np.ceil(original_size * 0.01)), \
-        "The scaled image does not have the expected dimensions!"
-
-
-def test_zero_scale_should_return_error():
-    img = vm.load_image("tests/img/raw/meme.jpg")
-    with pytest.raises(ValueError):
-        vm.scale(img, 0)
-
-
-def test_negative_scale_should_return_error():
-    img = vm.load_image("tests/img/raw/meme.jpg")
-    with pytest.raises(ValueError):
-        vm.scale(img, -1.0)
-
-
-def test_pillow_image_should_return_error():
-    img = Image.open("tests/img/raw/meme.jpg")
+def test_scale_with_invalid_scale_type_should_raise_error():
+    img = load_image('tests/img/raw/meme.jpg')
     with pytest.raises(TypeError):
-        vm.scale(img, 1.0)
\ No newline at end of file
+        scale(img, 'invalid_scale')
+        
+def test_scale_with_valid_image_should_not_raise_error():
+    img = load_image("tests/img/raw/meme.jpg")
+    try:
+        scale(img, 1.5)
+    except Exception as e:
```

Comment:
These tests are not testing for specific error types. Also, as we're using `pytest`, please use `pytest.raises()` which usages are outlined in the documentation [here](https://docs.pytest.org/en/7.4.x/how-to/assert.html#assertions-about-expected-exceptions) instead of built-in `try`/`catch`.